datasets/adv_glue/README.md

```diff
@@ -0,0 +1,56 @@
+---
+pretty_name: Adversarial GLUE
+languages:
+- en
+paperswithcode_id: adversarial-glue-a-multi-task-benchmark-for
+---
+
+# Dataset Card for "adv_glue"
+
+## Dataset Description
+
+Adversarial GLUE Benchmark (AdvGLUE) is a comprehensive robustness evaluation benchmark that focuses on the adversarial robustness evaluation of language models. It covers five natural language understanding tasks from the famous GLUE tasks and is an adversarial version of GLUE benchmark.
+
+AdvGLUE considers textual adversarial attacks from different perspectives and hierarchies, including word-level transformations, sentence-level manipulations, and human-written adversarial examples, which provide comprehensive coverage of various adversarial linguistic phenomena.
+
+### Supported Tasks and Leaderboards
+
+Leaderboard available on the homepage: [https://adversarialglue.github.io/](https://adversarialglue.github.io/).
+
+### Languages
+
+AdvGLUE deviates from the GLUE dataset, which has a base language of English.
+## Dataset Structure
+
+We show detailed information for up to 5 configurations of the dataset.
+
+### Data Instances
+
+#### default
+
+- **Size of downloaded dataset files:** 198 KB
```

Comment:
Could you provide an actual example from the dataset here ? Feel free to use `load_dataset` and take the first one

Reply:
done!